exporter/sapmexporter/exporter.go

```diff
@@ -92,13 +92,13 @@ func (se *sapmExporter) tracesByAccessToken(td pdata.Traces) map[string]pdata.Tr
 		accessToken := ""
 		if !resourceSpan.Resource().IsNil() {
 			attrs := resourceSpan.Resource().Attributes()
-			attributeValue, ok := attrs.Get(splunk.SFxAccessTokenLabel)
-			if ok {
-				attrs.Delete(splunk.SFxAccessTokenLabel)
```

Comment:
What problem was this causing?

Reply:
It is interesting. We return a pointer to a memory block in the array. When the element is deleted we reuse memory so will point to a different object.

Need to discuss this a bit more in core when I will file a PR to add a comment to the Get